rcl/test/rcl/test_arguments.cpp

```diff
@@ -1021,9 +1021,12 @@ TEST_F(CLASSNAME(TestArgumentsFixture, RMW_IMPLEMENTATION), test_param_argument_
   ASSERT_TRUE(NULL != param_value->bool_array_value);
   ASSERT_TRUE(NULL != param_value->bool_array_value->values);
   ASSERT_EQ(3U, param_value->bool_array_value->size);
-  EXPECT_TRUE(param_value->bool_array_value->values[0]);
-  EXPECT_FALSE(param_value->bool_array_value->values[1]);
-  EXPECT_FALSE(param_value->bool_array_value->values[2]);
+  bool bool_value = param_value->bool_array_value->values[0];
```

Comment:
Analysis complains about forming a reference to a nullptr here and below, which is actually impossible. I'm not sure why it only does it with bool arrays.